spec/services/idv/steps/verify_step_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe Idv::Steps::VerifyStep do
+  let(:user) { build(:user) }
+  let(:service_provider) do
+    create(:service_provider,
+           issuer: 'http://sp.example.com',
+           app_id: '123')
+  end
+
+  describe '#call' do
+    it 'sets uuid_prefix on pii_from_doc' do
+      controller = instance_double('controller',
+                                   session: { sp: { issuer: service_provider.issuer } },
+                                   current_user: user)
+      flow = Idv::Flows::DocAuthFlow.new(controller, {}, 'idv/doc_auth')
+      flow.flow_session = { pii_from_doc: {} }
+      step = Idv::Steps::VerifyStep.new(flow)
+      step.call
+
+      expect(flow.flow_session[:pii_from_doc][:uuid_prefix]).to eq service_provider.app_id
```

Comment:
Could we follow this one step further? Like something along the lines of this? Just to make sure the gems get the params we put inside this mystery `pii_from_doc` hash?


```ruby
expect_any_instance_of(Idv::Agent).
  to receive(:proof_resolution).with hash_including(uuid_prefix: service_provider.app_id)
```

Reply:
Or does that call happen outside this step?

Reply:
It does yeah, good call, added that